### PR TITLE
Mobile portfolio first page

### PR DESCRIPTION
**Pull request by Ol-create**
- Add navbar with styles.
- Add description texts with styles.
- Add social media icons.
- Add a background image to the first page.

### CODE REVIEW

**Review by keroloussamy — COMMENTED** [review]
Hi @Ol-create, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has linter errors, please make sure that your linter is passing and after that submit again.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by Amrhub** [comment]
Hi @Ol-create, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't meet the project requirement you might have mistaken and submitted it in the wrong dashboard but please make sure to submit it in the right project in your dashboard. because it's shown to me that your project should meet these [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) this where I think there is a mistake.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                <div class="bar2"></div>
+                <div class="bar3"></div>
+            </div>
+        </nav>
+    </header>
+    <main id="container">
+        <header id="greetText">
+            <h1>Hey There.<br>
+                I'm Oluyemi</h1>
+        </header>
```
Comment (CollinsTatang) [inline index.html]: - [ ] Kindly remove this `header` tag and put the heading inside the `section` tag.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        <section>
+            <h2 id="jobTitle">I am a Software Developer</h2>
+            <p id="description">I can help you build a product , feature or website Look through
+                some of my work and experience! If you like what you see and
+                have a project you need coded,
+                don’t hestiate to contact me.</p>
```
Comment (CollinsTatang) [inline index.html:37]: Great use of HTML semantics

File: index.html [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+        </section>
+        <ul class="icon">
+            <li><a href="https://twitter.com/OluyemiPaul99"><i class="fa fa-twitter fa-lg"></i> </a></li>
+            <li><a href="https://www.linkedin.com/in/paul-oluyemi-193966ab"><i class="fa fa-linkedin fa-lg"></i> </a>
+            </li>
+            <li><a href="#"><i class="fa fa-medium fa-lg"></i> </a></li>
+            <li><a href="https://github.com/Ol-create"><i class="fa fa-github fa-lg"></i> </a></li>
+            <li><a href="#"><i class="fa fa-angellist fa-lg"></i> </a></li>
+        </ul>
```
Comment (CollinsTatang) [inline index.html:46]: Great job general design is very good! All background images, margins, and details are really similar to Figma's!

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @Ol-create , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- PR is descriptive ✔️
- Gitflow flow was correctly used ✔️ 
- Descriptive README file ✔️


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 3 above)
Comment (Ol-create) [reply index.html:46]: Thank you for your effort. I appreciate you.

**Review by Whoistolu — APPROVED** [review]
### Hi @Ol-create,

Your project is complete!  There is nothing else to say other than... It's time to merge.

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**